jobs/rocoto/earc.sh

```diff
@@ -134,20 +139,26 @@ if [ $ENSGRP -eq 0 ]; then
             [ ! -d $ATARDIR/$CDATE ] && mkdir -p $ATARDIR/$CDATE
         fi
 
+        set +e
         $TARCMD -P -cvf $ATARDIR/$CDATE/enkf${CDUMP}.tar $(cat $ARCH_LIST/enkf${CDUMP}.txt)
         status=$?
         if [ $status -ne 0  -a $CDATE -ge $firstday ]; then
             echo "$(echo $TARCMD | tr 'a-z' 'A-Z') $CDATE enkf${CDUMP}.tar failed"
             exit $status
         fi
+        ${ERR_EXIT_ON:-set -eu}
     fi
 
     #-- Archive online for verification and diagnostics
     [[ ! -d $ARCDIR ]] && mkdir -p $ARCDIR
     cd $ARCDIR
 
-    $NCP $ROTDIR/enkf${CDUMP}.$PDY/$cyc/$COMPONENT/${CDUMP}.t${cyc}z.enkfstat         enkfstat.${CDUMP}.$CDATE
-    $NCP $ROTDIR/enkf${CDUMP}.$PDY/$cyc/$COMPONENT/${CDUMP}.t${cyc}z.gsistat.ensmean  gsistat.${CDUMP}.${CDATE}.ensmean
+    if [[ -f $ROTDIR/enkf${CDUMP}.$PDY/$cyc/$COMPONENT/${CDUMP}.t${cyc}z.enkfstat ]]; then
```

Comment:
why not use `safe_copy` here?

Reply:
Probably just missed it. Will update. I think we need to have someone review this and identify which files we should fatally fail on and which we can continue on without.

Reply:
Oh, it's not that I missed it. I fixed this bit; then later I realized it was going to keep being an issue and made a function instead, but never replaced the first edit with the function. Fixed now.

We could still use some feedback on which of these (if any) should be fatal. These changes are just ignoring missing files so it completes without regard to whether we _should_ or not.

Reply:
Wait, safe_copy isn't even in this script, it's in the other archive script. Should I copy that function here, even though it is only needed a few times in this script?

Reply:
perhaps put that in a `ush/workflow_utils.sh` and source it.  You could call it `file_utils.sh` if you wish to use other file/directory manipulations.

Reply:
Done. I'm not going to go through every script (yet) and replace all the file checking with safe_copy(), but did apply it to earc.

Reply:
Maybe I should rename it non_blocking_copy?

Reply:
Changed to nb_copy